e2e/test_module.py

```diff
@@ -97,5 +111,47 @@ def test_end2end(test_dir: Path, db: Memgraph):
         except Exception:
             assert True
 
+
+def _test_static(test_dir: Path, db: Memgraph):
+    """
+    Testing static modules.
+    """
+    input_cyphers = test_dir.joinpath(TestConstants.INPUT_FILE).open("r").readlines()
+    _execute_cyphers(input_cyphers, db)
+
+    test_dict = _load_yaml(test_dir.joinpath(TestConstants.TEST_FILE))
+    _run_test(test_dict, db)
+
+
+def _test_online(test_dir: Path, db: Memgraph):
+    """
+    Testing online modules. Checkpoint testing
+    """
+    checkpoint_input = _load_yaml(test_dir.joinpath(TestConstants.INPUT_FILE))
+    checkpoint_test_dicts = _load_yaml(test_dir.joinpath(TestConstants.TEST_FILE))
+
+    setup_cyphers = checkpoint_input[TestConstants.SETUP]
+    checkpoint_input_cyphers = checkpoint_input[TestConstants.QUERIES]
+    cleanup_cyphers = checkpoint_input[TestConstants.CLEANUP]
+
+    _execute_cyphers(setup_cyphers.splitlines(), db)
```

Comment:
This we could make to be optional, setup and cleanup. Maybe some tests don't need setup but need cleanup or something similar.